person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  attr_accessor :name, :age
+
+  def can_use_services?
+    @age > 18 || @parent_permission == true
```

Comment:
- [ ] [OPTIONAL] To not repeat yourself and keep the code DRY kindly make sure to use the `of_age?` instead of using `@age > 18`